web/src/locales/zh/data.json

```diff
@@ -14,11 +14,17 @@
   },
   "article": {
     "Abstract": "Abstract",
+    "Article added successfully": "Article added successfully",
```

Comment:
You should use the same i18n texts for all CRUD: https://github.com/casdoor/casdoor/blob/86c10fe0abbf4cbcfec2b75663c38a8a1de72ec5/web/src/locales/zh/data.json#L270-L276